app/services/custom-forms.js

```diff
@@ -2,91 +2,22 @@ import Ember from 'ember';
 
 const {
   Service,
-  computed,
   get,
   inject,
   isEmpty,
   set
 } = Ember;
 
 export default Service.extend({
-  currentModel: null,
-  customForms: {},
-  formsForType: null,
   store: inject.service(),
 
-  formsForSelect: computed('formsForType', 'usedForms', function() {
```

Comment:
All the possible caching was removed. Service now has a simple purpose and interface.